gwpy/signal/window.py

```diff
@@ -56,6 +56,8 @@ def canonical_name(name):
     >>> canonical_name('ksr')
     'kaiser'
     """
+    if name == 'planck':  # make sure to handle the Planck window function
```

Comment:
can you modify this to be

```python
    if name.lower() == 'planck':
```

to allow people to use `name='Planck'`?

Reply:
done in commit 21f24d31b156aa1cf435496a6d85b43c95efdaaf